blurb/blurb.py

```diff
@@ -956,9 +956,13 @@ def init_tmp_with_template():
             failure = str(e)
 
         if not failure:
-            assert len(blurb) # if parse_blurb succeeds, we should always have a body
-            if len(blurb) > 1:
-                failure = "Too many entries!  Don't specify '..' on a line by itself."
+            with open(tmp_path, "rt", encoding="utf-8") as file:
```

Comment:
The file seems to be already read in `blurb.load/parse`, so you should be able to access it (or even the issue number directly) directly from the instance without opening and parsing the file again.